### PR TITLE
Add number of mini cart items on the product add to cart

### PR DESCRIPTION
**Pull request by amit-webkul**

### CODE REVIEW

File: packages/Webkul/Shop/src/Resources/views/checkout/cart/mini-cart.blade.php [hunk 1]
```diff
@@ -176,7 +176,7 @@ class="m-0 ml-[0px] block mx-auto bg-navyBlue text-white text-base w-full font-m
                  * Action.
                  */
                 this.$emitter.on('test-cart', (cart) => {
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/checkout/cart/mini-cart.blade.php:178]: Event name should be appropriate

Comment (jitendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/mini-cart.blade.php:178]: Also, this event callback should accept cart not cart.cart

Comment (amit-webkul) [reply packages/Webkul/Shop/src/Resources/views/checkout/cart/mini-cart.blade.php]: Sir fixed

File: packages/Webkul/Shop/src/Http/Controllers/API/WishlistController.php [hunk 2]
```diff
@@ -109,16 +108,17 @@ public function moveToCart($id): JsonResource
             if ($result) {
                 return new JsonResource([
                     'data'     => WishlistResource::collection($this->wishlistRepository->get()),
+                    'cart'     => ['items_count'  => Cart::getCart()->items_count],
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/API/WishlistController.php:111]:
~~~php
...
'data' => ['wishlist' => [], 'cart' => []];
...
~~~

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**